notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1788-M2_bending_mode_actuation_repetability.ipynb

```diff
@@ -0,0 +1,525 @@
+   },
+   "source": [
+    "## Apply one Bending Modes to fine tune the amplitude/scaling factor\n",
+    "\n",
+    "The fine tuning of the bending mode scaling factor is done for groups of 10 bending modes at the time. This step can be skipped if the scaling factors are already defined."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "39977b6f-0cac-4d07-8de1-71300fe8d135",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "bending_mode = get_bending_mode()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "591d6163-47e6-4ae4-8817-b765587f0db6",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# bending modes vector indexes\n",
+    "decina = 0 #Select the tens-order of the bending mode, from 0 to 6  \n",
+    "idx = 0 #Select the bending mode inside the tens-order, from 0 to 9\n",
+    "\n",
+    "# genereate a vector of 10 elements to be populated with the ascending bending mode order \n",
+    "idx_bending_mode = [0] * 10\n",
+    "for idx in range(10):\n",
+    "    idx_bending_mode[idx] = idx + decina*10\n",
+    "    \n",
+    "print(idx_bending_mode)\n",
+    "# bending mode scaling factors, to be fine tuned, applied force shall be around 10 N to get 500 nm RMS surface deformation\n",
+    "amplitude = np.array((1, 0.1, 0.1, 0.05, 0.05, 0.01, 0.01, 0.01))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "980e168a-a70c-4b73-ad24-85c5d92050b4",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# bending mode number from 0 to 9\n",
+    "idx = 0\n",
+    "\n",
+    "print(f\"Bending mode is {idx_bending_mode[idx]} and amplitude is {amplitude[decina]}\")\n",
+    "forces = get_bending_mode_forces(bending_mode, idx_bending_mode[idx], amplitude[decina])\n",
+    "\n",
+    "# apply single bending mode\n",
+    "await apply_force_cycle_axial(m2, forces, time_forces=5, time_reset=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6344b357-b846-475b-b8e6-487b9676027a",
+   "metadata": {},
+   "source": [
+    "## Looping over multiple bending modes \n",
+    "Loop over multiple bending modes with positive and negative signs at groups of 10 bending modes. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "114ab86c-670a-43c7-a0ae-9fa799a31a3c",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "bending_mode = get_bending_mode()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2322d410-b397-466b-a5a4-b06d6a7d89dd",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# array definition\n",
+    "\n",
+    "# bending modes scaling factors\n",
+    "amplitude = np.array((1, 0.1, 0.1, 0.05, 0.05, 0.01, 0.01, 0.01))\n",
+    "\n",
+    "# index scaling \n",
+    "idx_scaling = np.array((0, 1, 2, 3, 4 ,5 ,6 ,7))"
```

Comment:
Not sure I understand what “idx_scaling” is for. I seems unused.

Reply:
you are correct, it was a leftover of a draft version, we removed it